src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -152,23 +152,23 @@ public function setAllowOverwrite($allow)
     }
 
     /**
-     * Sets the closures used for normalization.
+     * Sets the callbacks used for normalization.
      *
-     * @param \Closure[] $closures An array of Closures used for normalization
+     * @param callable[] $callbacks An array of callbacks used for normalization
      */
-    public function setNormalizationClosures(array $closures)
+    public function setNormalizationCallbacks(array $callbacks)
```

Comment:
To make it perfectly clear, we don't want to break BC without providing an upgrade path. In this case, we should introduce the new method in 2.8, deprecate the old one in 2.8, and remove the deprecated on in 3.0.